go.mod

```diff
@@ -2,9 +2,21 @@ module github.com/k8sgpt-ai/k8sgpt-operator
 
 go 1.21
 
+replace github.com/k8sgpt-ai/k8sgpt-operator/api/v1alpha1 => /Users/ronaldpetty/github.com/ronaldpetty/k8sgpt-operator/api/v1alpha1
```

Comment:
not sure why you needed to replace in go.mod  @ronaldpetty :thinking:

Reply:
This was probably to modify the CRD to add kyverno to the enum?

Reply:
Yes, this is correct. There might be an easier way, not an expert here.